pkg/node/manager/manager.go

```diff
@@ -133,7 +141,9 @@ func (m *manager) Subscribe(nh datapath.NodeHandler) {
 	m.mutex.RLock()
 	for _, v := range m.nodes {
 		v.mutex.Lock()
-		nh.NodeAdd(v.node)
+		if err := nh.NodeAdd(v.node); err != nil {
+			log.WithError(err).Errorf("handler initial subscribe: %s failed on node %s", nh.Name(), v.node.Name)
```

Comment:
For error and warning level logs, we need to be more thoughtful about the log msgs. The msgs need to be useful for the end user by telling them what are the ramifications and what Cilium will do about it. In some cases there's nothing that a user can do and so for example, the msg can say to file an issue.

In this case, I'd suggest 

```
Initial node event handling failed. Cilium may be in a degraded state. See error message for more details.
```

In the future, we can improve the error msg to say that Cilium will retry, etc. 

Please apply the above to the rest of the PR for errors / warnings logs.